packages/core/core.ts

```diff
@@ -1,4 +1,4 @@
-import { ComponentType, StatelessComponent, createElement, forwardRef } from 'react'
+import { ComponentType, FC, createElement, forwardRef } from 'react'
```

Comment:
`StatelessComponent` type is deprecated. Now we should use `FC`

https://stackoverflow.com/questions/44133420/what-is-the-typescript-return-type-of-a-react-stateless-component